nipoppy/env.py

```diff
@@ -9,6 +9,7 @@
 # BIDS
 BIDS_SUBJECT_PREFIX = "sub-"
 BIDS_SESSION_PREFIX = "ses-"
+FAKE_SESSION = "ses-IsFake"
```

Comment:
For consistency with the rest of the codebase and how we differentiate between session IDs with and without the BIDS prefix, I think this should be either `FAKE_SESSION_ID = "IsFake"` (or whatever name we decide to name this) or `FAKE_BIDS_SESSION_ID = "ses-IsFake"`. Would prefer `FAKE_SESSION_ID = "IsFake"` since most of the code doesn't use the BIDS prefix